### PR TITLE
propocessing about geojson

### PR DESCRIPTION
**Pull request by maimuderizi**
.

### CODE REVIEW

File: src/main/resources/propocessing/GeoJsonLoader.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	private static GeometryFactory geometryFactory = JTSFactoryFinder
+			.getGeometryFactory(null);
+
+	public static void main(String[] args) throws IOException {
+		// TODO Auto-generated method stub
+		String path = System.getProperty("user.dir");
+		List<City> lc = new ArrayList<City>();
+		for (int i = 1; i < 80; i++) {
+			String str = String.format("%02d", i);
+			readCity("D:/testGeo/" + str + ".json", lc);
```
Comment (JavierJia) [inline src/main/resources/propocessing/GeoJsonLoader.java]: Change to a relative path?

**Comment by JavierJia** [comment]
@maimuderizi Very nice work! :+1: 
One general comments, can you rename these 01.json 02.json .... to meaningful names?

**Comment by maimuderizi** [comment]
bug fix
===========
ok.
(it maybe a little inconvenient because each *.json has being such name since it was downloaded from TIGER separately.

**Comment by maimuderizi** [comment]
1.rename .json files
2.add property : "ID"
3.split multipolygon to multiple single polygon

File: .classpath [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (JavierJia) [inline .classpath]: should be removed.

File: .project [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (JavierJia) [inline .project]: ditto

File: pom.xml [hunk 4]
```diff
@@ -1,113 +1,123 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (JavierJia) [inline pom.xml]: could you change the indent to 4 space?

File: src/.metadata/.plugins/org.eclipse.rse.core/profiles/PRF.ubuntu_0/FP.local.files_0/node.properties [hunk 5]
```diff
@@ -0,0 +1,57 @@
+# RSE DOM Node
```
Comment (JavierJia) [inline src/.metadata/.plugins/org.eclipse.rse.core/profiles/PRF.ubuntu_0/FP.local.files_0/node.properties]: please remove all the .* configuration files.

**Comment by JavierJia** [comment]
@maimuderizi  Very nice work! Thanks!
Could you add some test cases as well?

**Pull request merged**